tests/general/pio_iosystem_tests2.F90.in

```diff
@@ -220,11 +220,14 @@ PIO_TF_AUTO_TEST_SUB_BEGIN three_files_two_iosystems_odd_even
                     pio_file, fname, attname, dimname, .true., ret)
     PIO_TF_CHECK_ERR(ret, "Checking contents of file failed :" // fname)
 
-    ! Make sure that we can still check the contents of the file
-    call check_file(odd_even_comm, odd_even_iosys, iotypes(i), pio_file, &
-                    fname, attname, dimname, ret)
-    PIO_TF_CHECK_ERR(ret, "Checking (second) contents of file failed :" // fname)
-
+    ! ! Make sure that we can still check the contents of the file
+    ! call check_file(odd_even_comm, odd_even_iosys, iotypes(i), pio_file, &
+    !                 fname, attname, dimname, ret)
+    ! PIO_TF_CHECK_ERR(ret, "Checking (second) contents of file failed :" // fname)
+
+    PIO_TF_LOG(0,*) "About to call MPI_Barrier"
+!    call  MPI_Barrier(pio_tf_comm_);
+    PIO_TF_LOG(0,*) "After MPI_Barrier"
```

Comment:
OK, thanks for catching that! I made some changes to that file while debugging. Now I have merged in the pio_iosystem_tests2.F90.in from the master, and it works without modification. So that's good. ;-)